### PR TITLE
Add Flutter 3.16 compatibility

### LINKED ISSUE #1: Flutter 3.16 support
**Describe the bug**
The SDK language version in Flutter 3.16 was updated to 3.2.0. To support Flutter 3.16, the analyzer package in Envy should be upgraded to the latest version.

**To Reproduce**
Try to use the `envy_generator: 0.0.4` package dependency with `analyzer: 6.2.0`.

### PR DESCRIPTION
**Pull request by estevez-dev**
Fixes https://github.com/ksokolovskyi/envy/issues/1.

### CODE REVIEW

File: envy_generator/pubspec.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 environment:
-  sdk: '>=2.17.0 <3.0.0'
+  sdk: ">=3.0.1 <4.0.0"
 
 dependencies:
-  analyzer: ">=5.0.0 <6.0.0"
+  analyzer: "^6.0.0"
   build: ^2.0.0
   dotenv: ^4.0.1
   envy_annotation: ^0.0.1
```
Comment (ksokolovskyi) [inline envy_generator/pubspec.yaml]: @estevez-dev Could you please update `envy_annotation` version constraint to the `^0.0.3`?
Version solving will fail, but after publishing to the pub, all will be fine.

**Comment by ksokolovskyi** [comment]
@estevez-dev Thanks a lot for the PR!
I left one comment in `envy_generator/pubspec.yaml`, and could you please change version of `build_runner` in `envy_generator/example/pubspec.yaml` to `^2.0.0`, because `dart pub get` complains about it?

**Comment by estevez-dev** [comment]
> @estevez-dev Thanks a lot for the PR! I left one comment in `envy_generator/pubspec.yaml`, and could you please change version of `build_runner` in `envy_generator/example/pubspec.yaml` to `^2.0.0`, because `dart pub get` complains about it?

It is done

**Comment by ksokolovskyi** [comment]
@estevez-dev, thanks for your contribution!
I've published the changes to the pub.dev, please check.

**Comment by estevez-dev** [comment]
@ksokolovskyi, everything is fine. Thanks

**Comment by ksokolovskyi** [comment]
@estevez-dev, thanks. 
Have a great day!

**Pull request merged**